_data/settings/utilities.yml

```diff
@@ -108,8 +108,8 @@ contents:
   - name: "Utility manual values"
     description: |
       Utilities output with the tokens and values set in the manual values map.
-    var: $[utility-module]-palettes
-    default: '"palette-[utility-module]-manual-values"'
+    var: $[utility-module]-manual-values
```

Comment:
A few legacy typos above `align-tems`.

Reply:
Good eye. Typos fixed in https://github.com/uswds/uswds-site/pull/2016/commits/840cfecb8e98b2ffcf0dbebf32f739a622aa1a4c